custom_components/extron/__init__.py

```diff
@@ -1,7 +1,5 @@
 """The Extron integration."""
 
-from __future__ import annotations
```

Comment:
This is only needed for python <=3.10

Reply:
Yeah, it was part of the `python3 -m script.scaffold integration` boilerplate (see https://developers.home-assistant.io/docs/creating_component_index)